### DISCUSSION TITLE
[Feature request] Mac OS Spotlight like

### DISCUSSION THREAD

**Discussion by mahfoudfx** [themesberg/flowbite#267] [Ideas]
Spotlight like feature is handy in Admin Panel to quick access for features.
You can find an exemple on [Tailwind UI Command Palettes](https://tailwindui.com/components/application-ui/navigation/command-palettes).

![windows10launcherpowertoy](https://user-images.githubusercontent.com/31103481/187665179-c026b5fe-8b3e-448d-ab4e-49c27d28ae7d.jpg)

**Comment by RobyCigar** [comment]
im thinking the same man
